components/TrackingComplaint/Accordion.vue

```diff
@@ -2,10 +2,12 @@
   <details
     class="complaint-accordion bg-green-50 p-4 rounded-[18px] mt-2"
     v-bind="$attrs"
+    :open="isOpen"
     v-on="$listeners"
   >
     <summary
       class="complaint-accordion__summary min-w-0 w-full flex justify-between"
+      @click.prevent="toggleAccordion"
     >
```

Comment:
minor/discuss:

hello @rayfajars I don't think we need to bind state with `details` and `summary` element, because it already handle toggling dropdown natively. However we still do need to handle to rotation of the icon using CSS

```css
details[open] #my-icon {
  // handle rotation
}
```

Reply:
thanks for suggest bli @Ibwedagama 

fix [here](https://github.com/jabardigitalservice/super-app-web/pull/114/commits/526980b3107c8ef03f475daa2125a1e6cd77a67e)